go.mod

```
@@ -1,6 +1,6 @@
 module github.com/sigstore/sigstore
 
-go 1.21
+go 1.22.0
```

```suggestion
go 1.22.4
```
can we bump to that? of just 1.22

---

IIRC you can't do `1.22` anymore without also adding `toolchain`, or you need to specify the specific go version like here.